### ISSUE TITLE
Double check comments handling

### ISSUE THREAD

**Issue by adamcik** [mopidy/mopidy#943]
Turns out there is a setting in MPD that sets if comments should be extracted. Given the recent removal of comments we should double check if this was right by looking at MPD behavior with that setting enabled.

**Comment by daveola** [comment]
Absolutely.  To see comments again, add 'comment' to the comma separated list of 'metadata_to_use' (and uncomment the setting, if needed) in mpd.conf, then update the library.

You can see success by playing a song that has a comment tag (such as 'Note' in id3v1) and calling:

mpc -f "Comment is %comment%"
